config/config.exs

```diff
@@ -5,4 +5,4 @@ use Mix.Config
       {"localhost", 9093},
       {"localhost", 9094},
     ],
-    consumer_group: false
+    consumer_group: "kafka_ex_dev"
```

Comment:
this will break for users using kafka < 0.8.2, see https://github.com/kafkaex/kafka_ex/commit/214cac1881883dd3ba85c075fbc76bda7d66dafa